charts/linstor-cluster/templates/_helpers.tpl

```diff
@@ -0,0 +1,73 @@
+{{/*
+Expand the name of the chart.
+*/}}
+{{- define "linstor-cluster.name" -}}
+{{- default .Chart.Name .Values.nameOverride | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Create a default fully qualified app name.
+We truncate at 63 chars because some Kubernetes name fields are limited to this (by the DNS naming spec).
+If release name contains chart name it will be used as a full name.
+*/}}
+{{- define "linstor-cluster.fullname" -}}
+{{- if .Values.fullnameOverride }}
+{{- .Values.fullnameOverride | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- $name := default .Chart.Name .Values.nameOverride }}
+{{- if contains $name .Release.Name }}
+{{- .Release.Name | trunc 63 | trimSuffix "-" }}
+{{- else }}
+{{- printf "%s-%s" .Release.Name $name | trunc 63 | trimSuffix "-" }}
+{{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Create chart name and version as used by the chart label.
+*/}}
+{{- define "linstor-cluster.chart" -}}
+{{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
+{{- end }}
+
+{{/*
+Common labels
+*/}}
+{{- define "linstor-cluster.labels" -}}
+helm.sh/chart: {{ include "linstor-cluster.chart" . }}
+{{ include "linstor-cluster.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/managed-by: {{ .Release.Service }}
+{{- end }}
+
+{{/*
+Selector labels
+*/}}
+{{- define "linstor-cluster.selectorLabels" -}}
+app.kubernetes.io/name: {{ include "linstor-cluster.name" . }}
+app.kubernetes.io/instance: {{ .Release.Name }}
+{{- end }}
+
+{{/*
+Return true, if apiTLS enabled and cert published via cert-manager
+*/}}
+{{- define "linstor-cluster.createApiTLSCert" -}}
+{{- if .Values.linstorCluster.apiTLS }}
+    {{- if .Values.linstorCluster.apiTLS.certManager }}
+        {{- true -}}
+    {{- end }}
+{{- end }}
+{{- end }}
+
+{{/*
+Return true, if internalTLS enabled and cert published via cert-manager
+*/}}
+{{- define "linstor-cluster.createInternalTLSCert" -}}
+{{- if .Values.linstorCluster.internalTLS }}
+    {{- if .Values.linstorCluster.internalTLS.certManager }}
+        {{- true -}}
+    {{- end }}
+{{- end }}
+{{- end }}
```

Comment:
Would there be any point in also supporting the use of certificates generated by Helm itself? With [`genCA`](https://docs.helm.sh/docs/chart_template_guide/function_list/#genca) and [`genSignedCert`](https://docs.helm.sh/docs/chart_template_guide/function_list/#gensignedcert)?